### PR TITLE
[PAN-3978] Adding SSH Key Management to THG-Client

### PR DESCRIPTION
**Pull request by andy-lu-42**
@billyspringsteen When releasing this, we mustn't create a release branch! The way this release is recognized by packagist, is by merging it into the main branch and then create a release (https://github.com/UK2group/thghosting-cilent/releases/new) with a new tag, in this case tag "1.3.1".

After merge, the branch must be deleted, if not deleted automatically.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "thg/thg-client",
     "description": "THG Open API client",
     "type": "library",
-    "version": "1.3.0",
+    "version": "1.3.1",
```
Comment (peter-uhlich) [inline composer.json:5]: @andy-lu-42 You need to bump the version whenever changes are made to repository

File: tests/Unit/ThgHosting/ThgHostingClientTest.php
Comment (peter-uhlich) [inline tests/Unit/ThgHosting/ThgHostingClientTest.php:1]: Providing unit tests is never a bad idea

**Review by peter-uhlich — APPROVED** [review]
Next time please give your branch a more meaningful name because it's visible public! See https://packagist.org/packages/thg/thg-client

**Comment by andy-lu-42** [comment]
I was trying to rename the branch but it seems to created a duplicate branch on packagist instead. and this PR got closed accidentally. the new PR is https://github.com/UK2group/thghosting-cilent/pull/20

**Pull request closed without merging**